src/atomate2/vasp/sets/mp.py

```diff
@@ -0,0 +1,123 @@
+"""
+Module defining jobs for Materials Project r2SCAN workflows.
+
+Reference: https://doi.org/10.1103/PhysRevMaterials.6.013801
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+import numpy as np
+from monty.serialization import loadfn
+from pkg_resources import resource_filename
+
+from atomate2.vasp.sets.base import VaspInputGenerator
+
+if TYPE_CHECKING:
+    from pymatgen.core import Structure
+    from pymatgen.io.vasp import Outcar, Vasprun
+
+
+_BASE_MP_R2SCAN_RELAX_SET = loadfn(
+    resource_filename("atomate2.vasp.sets", "BaseMPR2SCANRelaxSet.yaml")
+)
+
+
+_BASE_MP_GGA_RELAX_SET = loadfn(
+    resource_filename("atomate2.vasp.sets", "BaseMPGGASet.yaml")
+)
+
+
+@dataclass
+class MPGGARelaxGenerator(VaspInputGenerator):
```

Comment:
A few comments:
- The standard notation is for input set generators to have `SetGenerator` at the end. So this (and the other generator in this file) would become `MPGGARelaxSetGenerator`. It is verbose but consistent with the other generators.
- I think rather than defining this whole class, you can just do:

```python
@dataclass
class MPGGARelaxSetGenerator(RelaxSetGenerator):
    config_dict: dict = field(default_factory=lambda: _BASE_MP_GGA_RELAX_SET)
```

Note that we are subclassing the default RelaxSetGenerator. Everything should then work out of the box.

Similarly, you can make a static set generator using:
```python
@dataclass
class MPGGAStaticSetGenerator(StaticSetGenerator):
    config_dict: dict = field(default_factory=lambda: _BASE_MP_GGA_RELAX_SET)
```

Reply:
Where does `auto_ismear` come into this? Like so?

```py

@dataclass
class MPGGARelaxSetGenerator(RelaxSetGenerator):
    """Class to generate MP-compatible VASP GGA relaxation input sets."""

    config_dict: dict = field(default_factory=lambda: _BASE_MP_GGA_RELAX_SET)

    def __post_init__(self):
        self.auto_ismear = False
        return super().__post_init__()
```

Reply:
Even simpler, just:
```
@dataclass
class MPGGARelaxSetGenerator(RelaxSetGenerator):
    """Class to generate MP-compatible VASP GGA relaxation input sets."""

    config_dict: dict = field(default_factory=lambda: _BASE_MP_GGA_RELAX_SET)
    auto_ismear: bool = False
```